test/.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "rules": {
+    "no-invalid-this": 0,
+    "no-console": 0,
```

Comment:
I removed all console calls from tests and made a test that always fails instead.